src/Agent.Sdk/Util/StringUtil.cs

```
@@ -293,5 +293,28 @@ public static string DeactivateVsoCommands(string input)
 
             return Regex.Replace(input, "##vso", "**vso", RegexOptions.IgnoreCase);
         }
+
+        public static string DeactivateBase64EncodedVsoCommands(string input)
+        {
+            if (string.IsNullOrEmpty(input))
+            {
+                return string.Empty;
+            }
+
+            try
+            {
+                byte[] decodedBytes = Convert.FromBase64String(input);
+                string decodedString = Encoding.UTF8.GetString(decodedBytes);
+
+                decodedString = DeactivateVsoCommands(decodedString);
+
+                byte[] bytes = Encoding.UTF8.GetBytes(decodedString);
+                return Convert.ToBase64String(bytes);
+            }
+            catch
```

Log the exception and throw it. Do not eat the exception without leaving any trace of it. A util method should leave it up to the caller to decide what to do with the exception.

This implementation leaves no trace of the exception and makes debugging (or even figuring out that there is an exception) very challenging